TriangleApp/app/src/main/java/triangle/triangleapp/helpers/MediaHelper.java

```diff
@@ -0,0 +1,192 @@
+package triangle.triangleapp.helpers;
+
+import android.hardware.Camera;
+import android.media.CamcorderProfile;
+import android.media.MediaRecorder;
+import android.os.Environment;
+import android.util.Log;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import triangle.triangleapp.view.CameraPreview;
+
+/**
+ * Created by danie on 9-6-2017.
+ */
+
+public class MediaHelper {
+    private final String TAG = "MediaHelper";
+    private boolean isRecording;
+    private MediaRecorder mMediaRecorder;
+    private CameraPreview mCameraPreview;
+    private Camera mCamera;
+    private final int MEDIA_TYPE_IMAGE = 1;
+    private final int MEDIA_TYPE_VIDEO = 2;
```

Comment:
modified to private static final String MEDIA_TYPE_IMAGE = 1; (const)
modified to private static final String MEDIA_TYPE_VIDEO = 2; (const)